app/util/data_preparation/api/bitbucket_clients.py

```diff
@@ -23,6 +33,7 @@ def get_entities(self, entity_name, batch_size, filter=None, max_results=500):
             start_at = start_at + len(response.json()['values'])
             loop_count -= 1
             if loop_count == 1:
+                # TODO variable max_results is not used, before delete it we should check if there is no bug in the code
```

Comment:
pls create a Jira ticket and add an id to this TODO

Reply:
I am going to investigate it today